### PR TITLE
Creating a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by DharmitD**
Signed-off-by: Dharmit Dalvi <ddalvi@redhat.com>

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [X] JIRA link(s): https://issues.redhat.com/browse/RHODS-3584
- [ ] The Jira story is acked
- [ ] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)
- [ ] The developer has manually tested the changes and verified that the changes work.

### CODE REVIEW

**Review by lucferbux — APPROVED** [review]
/lgtm

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Please take a few minutes to read GitHub's guide on [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/).
+It's a quick read, and it's a great way to introduce yourself to how things work behind the scenes in open-source projects.
+
+### Documentation
+
+If you want to update documentation, [README.md](README.md) is the file you're looking for.
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+### How to contribute code to odh-deployer
```
Comment (anishasthana) [inline CONTRIBUTING.md:16]: Can you add a note here saying who the primary QE contact is for the repo?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- Configure name and email in git
+- Fork this repo
+- In your fork, create a branch for your feature
+- Sign off your commit using the -s, --signoff option. Write a good commit message (see [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/))
+- Push your changes
+- Send a PR to odh-deployer using GitHub's web interface
+
+### Testing the PR
```
Comment (anishasthana) [inline CONTRIBUTING.md:25]: Can you add a note here on quick testing? When I'm quickly iterating on changes what I like doing is manually running the deploy script from the terminal.  This way I don't waste time building a new image every time. Definitely helps in that initial rapid iteration phase.

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (DharmitD) [reply CONTRIBUTING.md:16]: Updated the doc to include names of primary QE contacts.

File: CONTRIBUTING.md (hunk 2 above)
Comment (DharmitD) [reply CONTRIBUTING.md:25]: Updated the doc to mention this process of testing manually.

**Comment by DharmitD** [comment]
Also updated the PR template to remove check box for adding an entry to the build doc, as we no longer require an entry to be created in the build doc.

**Review by anishasthana — APPROVED** [review]

**Pull request merged**